aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -9,3 +47,192 @@
 #else
 #include <ATen/native/cuda/ROCmLoops.cuh>
 #endif
+
+namespace at { namespace native {
```

Comment:
Moved from `CUDALoops.cuh` and `ROCmLoops.cuh`, this part of the code is identical for CUDA and ROCm.